index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Atleti</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" type="text/css" href="//https://fonts.googleapis.com/css2?family=Outfit" />
+</head>
+<header class="main-header">
+
+    <div class="logo">
+        <img src="atletilogo.png" width="200" height="200" alt="AtletiLOGO">
```

Comment:
Really good to see that you added `alt` text. I would say that the format could be improved by separating the words by a space to make it more descriptive.